modules/homestakeros/options.nix

```diff
@@ -320,6 +320,28 @@
       };
 
       addons = {
```

Comment:
This probably needs an enable option, no?

Reply:
The UI does not have to be fully faithful of the the backend.

But let it be, I think this makes an interesting test case for adding software "on the fly" without kexec.